docs/native/src/scenarios/from_app.rst

```diff
@@ -1,3 +1,4 @@
+..  _OpenFilesiOSOffice:
```

Comment:
You can use spaces in the identifiers, and I suggest keeping them short, e.g. `open files`